packages/twenty-server/src/engine/twenty-orm/storage/data-source.storage.ts

```diff
@@ -0,0 +1,17 @@
+import { DataSource } from 'typeorm';
+
+export class DataSourceStorage {
```

Comment:
What's the impact on performance if we have many datasources with opened connections? Should we provide TTL and destroy the datasource after some time?

Reply:
Actually it's storing data source the same way it was done before, but we can of course optimize this with TTL if needed, but I'm not sure it's really needed

Reply:
Sure, let's discuss later if we need to improve that part 👍